### DISCUSSION TITLE
报错，怎么解决？

### DISCUSSION THREAD

**Discussion by happyleo80** [datawhalechina/DOPMC#136] [问题求助] [answered]
https://github.com/datawhalechina/prompt-engineering-for-developers/blob/main/content/2.%20%E6%8F%90%E7%A4%BA%E5%8E%9F%E5%88%99%20Guidelines.ipynb

一直出现报错，
AuthenticationError: No API key provided. You can set your API key in code using 'openai.api_key = <API-KEY>', or you can set the environment variable OPENAI_API_KEY=<API-KEY>). If your API key is stored in a file, you can point the openai module at it with 'openai.api_key_path = <PATH>'. You can generate API keys in the OpenAI web interface. See https://platform.openai.com/account/api-keys for details.

**Comment by Sm1les** [comment]
@happyleo80 同学你好，你可以直接去他们repo的issue区提问哈，地址是：https://github.com/datawhalechina/prompt-engineering-for-developers/issues

**Comment by hscspring** [comment, marked as answer]
> https://github.com/datawhalechina/prompt-engineering-for-developers/blob/main/content/2.%20%E6%8F%90%E7%A4%BA%E5%8E%9F%E5%88%99%20Guidelines.ipynb
> 
> 一直出现报错， AuthenticationError: No API key provided. You can set your API key in code using 'openai.api_key = ', or you can set the environment variable OPENAI_API_KEY=). If your API key is stored in a file, you can point the openai module at it with 'openai.api_key_path = '. You can generate API keys in the OpenAI web interface. See https://platform.openai.com/account/api-keys for details.

需要你换成自己的key
